pip/qsharp/telemetry_events.py

```diff
@@ -85,3 +85,42 @@ def on_estimate_end(durationMs: float, qubits: Union[str, int]) -> None:
         properties={"qubits": get_qubits_bucket(qubits)},
         type="histogram",
     )
+
+
+# Qiskit telemetry events
+
+
+def on_qiskit_import() -> None:
+    log_telemetry("qiskit.import", 1)
+
+
+def on_qiskit_run(shots: int, num_circuits: int) -> None:
+    log_telemetry(
+        "qiskit.run",
+        1,
+        properties={"shots": get_shots_bucket(shots), "num_circuits": num_circuits},
```

Comment:
What is the range for 'num_circuits'? Properties on metrics should be a limited set of value (which is why we bucket shots and qubits). Could this end up being dozens (or more) of distinct values?